### PR TITLE
runtime: Fix typo

### PR DESCRIPTION
**Pull request by rst0git**

### CODE REVIEW

File: pkg/adapter/runtime.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/containers/libpod/pkg/util"
 	"github.com/containers/storage/pkg/archive"
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (rhatdan) [inline pkg/adapter/runtime.go]: Why is this change required?

Comment (rst0git) [reply pkg/adapter/runtime.go]: The format-on-save feature of my editor automatically added it but it is not required or necessary. I removed it.
